_episodes/running-containers.md

```diff
@@ -184,11 +186,12 @@ command and they will execute inside the running container.
 ## Running containers interactively
 
 In all the examples above, Docker has started the container, run a command, and then
-immediately shut down the container. But what if we wanted to keep the container
+immediately stopped the container. But what if we wanted to keep the container
 running so we could log into it and test drive more commands? The way to
-do this is by adding the interactive flag `-it`
+do this is by adding the interactive flags `-it`
```

Comment:
Do we want to add a note here that for flags preceded by a single hyphen/minus, multiple flags can be combined after a single minus - as such, `-it` is the combination of the `-i` flag for interactive and the `-t` flag for _tty_, combined these flags make it possible for you to interact with the terminal in a running container.

Reply:
Done in updated version